packages/manager/modules/vps/src/migration/components/plan/plan.constants.js

```diff
@@ -0,0 +1,4 @@
+export const MIGRATE_MAX_DATE = '30.06.2020';
```

Comment:
Should be `27/07/2020` I guess

Reply:
no its, 30/06/2020. This date need to be taken from API. GET migrate2014 has this data.

Reply:
looks like this is not used, check and remove